ranger-hub-server-bundle/src/main/java/io/appform/ranger/hub/server/bundle/configuration/RangerHttpUpstreamConfiguration.java

```diff
@@ -13,30 +13,36 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.appform.ranger.http.server;
+package io.appform.ranger.hub.server.bundle.configuration;
+
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import io.appform.ranger.http.server.bundle.config.RangerHttpConfiguration;
-import io.dropwizard.Configuration;
-import lombok.AllArgsConstructor;
+import io.appform.ranger.http.config.HttpClientConfig;
+import io.appform.ranger.hub.server.bundle.models.BackendType;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
+import lombok.ToString;
 
 import javax.validation.Valid;
 import javax.validation.constraints.NotEmpty;
-import javax.validation.constraints.NotNull;
+import java.util.List;
 
 @Data
-@AllArgsConstructor
-@NoArgsConstructor
 @EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class HttpAppConfiguration extends Configuration {
-    @NotEmpty
-    @NotNull
-    String name;
-    @Valid
-    @NotNull
-    RangerHttpConfiguration rangerConfiguration;
+public class RangerHttpUpstreamConfiguration extends RangerUpstreamConfiguration {
```

Comment:
Can use SuperBuilder and get away with the constructor.

Reply:
we need to send type super ...